new_platforms/docs/win_sgx_dev.md

```diff
@@ -1,114 +1,196 @@
+WDK, these can be found
+[here](https://docs.microsoft.com/en-us/windows-hardware/drivers/other-wdk-downloads).
 
 # Working with the SDK
 
 ## Cloning the SDK
 
-This project uses submodules to pull and keep track of external dependencies. 
-Some of these recursively have submodules themselves. 
-When cloning, be sure to fetch all submodules recursively, and to update submodules
-recursively when you pull, too:
+This project uses submodules to pull and keep track of external dependencies.
+Some of these recursively have submodules themselves. When cloning, be sure to
+fetch all submodules recursively, and to update submodules recursively when you
+pull, too:
 
 ```
 git clone https://github.com/Microsoft/openenclave --recurse-submodules -b feature.new_platforms
 ```
 
 ## Building
 
-To build for Windows open the following solution with Visual Studio 2015 or 2017.
+To build for Windows, open the following file with Visual Studio 2017 via File
+-> Open -> CMake...:
 
 ```
- openenclave\new_platforms\OpenEnclave.sln
- ```
-
-The solution contains all libraries and samples needed to build the SDK. 
-Build for the chosen platform (x86, x64, or ARM) and configuration (Debug, DebugSimulation,
-or DebugOpteeSimulation).
-
-### Note on configuration support
-
-For this preview, the SDK does not fully support building SGX enclaves targeted for release.
-The configuration options supported allow for debug builds that can run on real hardware and
-debug simulated builds that can be used as part of your development cycle.
-
-**Note:** The solution can build ARM Hosts for Windows running in the REE. This preview does not have complete support for this yet, but it is coming soon. Building for ARM requires the [Windows Driver Kit for ARM](https://docs.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk).
+openenclave\new_platforms\CMakeLists.txt
+```
 
-| Configuration        | Platform    | Usage                                                                            |
-| -------------------- | ----------- | -------------------------------------------------------------------------------- |
-| DebugSimulation      | x86/x64     | Builds an SGX simulated enclave and x86 or x64 host app                             |
-| DebugSimulation      | ARM         | Builds a OP-TEE simulated enclave and ARM host app                               |
-| DebugOpteeSimulation | x86/x64/ARM | Builds a OP-TEE simulated enclave and x86, x64, or ARM host app                       |
-| Debug                | x86/x64     | Builds an SGX enclave and x86 or x64 host app                                       |
-| Debug                | ARM         | An ARM Windows host app. A non-simulated OP-TEE enclave must be built from Linux  (e.g., a bash on Ubuntu on Windows shell). |
+The solution contains all libraries and samples needed to build the SDK. From
+the configurations drop-down in the top toolbar in Visual Studio, select the one
```

Comment:
change "configurations" to "configuration's" (with apostrophe)

Reply:
I find the grammar with the possessive somewhat awkward, implying that the drop-down belongs to a configuration. Perhaps "configuration drop-down" in singular form?